### ISSUE TITLE
Large amount of artifacting in images.

### ISSUE THREAD

**Issue by ghost** [DavidBuchanan314/ambiguous-png-packer#5]
It seems if the input images are fairly detailed the output image ends up with a lot of artifacting, nearly to the point of destroying the image depending on the input. Is this a side effect of the exploit or a bug / something on my end? I notice your example images have _some_ artifacting, but not nearly as much as I'm seeing
![output2](https://user-images.githubusercontent.com/37161147/146788377-ec7a2d10-8b6e-4097-871b-727ba09cf448.png)
.

**Comment by DavidBuchanan314** [comment]
Yes, it's inherent in how the exploit works. You can reduce the artefacting by making your images more compressible. You can do this by:

- Upscaling with nearest-neighbor interpolation
- Reducing the number of colours in the image

I've been meaning to add this to the readme

To explain further, the artefacts that you see are actually the compressed bytes of the secondary image. So the more compression ratio you have, the better your artefact-to-image ratio is.
